app/views/root/_publication.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if publication.published? %>
       <%= link_to "#{Plek.new.website_root}/#{publication.slug}", class: 'link-muted' %>
-    <% elsif publication.safe_to_preview? %>
+    <% elsif !publication.archived? %>
```

Comment:
OK, so I looked at `content_models` and it wasn't that bad (and probably best in the long run), so here's https://github.com/alphagov/govuk_content_models/pull/417.